bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/i18n/boschshc.properties

```diff
@@ -76,6 +83,13 @@ channel-type.boschshc.combined-rating.description = Combined rating of the air q
 channel-type.boschshc.combined-rating.state.option.GOOD = Good Quality
 channel-type.boschshc.combined-rating.state.option.MEDIUM = Medium Quality
 channel-type.boschshc.combined-rating.state.option.BAD = Bad Quality
+channel-type.boschshc.communication-quality.label = Communication Quality
+channel-type.boschshc.communication-quality.description = The quality of communication between a device and the Bosch Smart Home Controller.
+channel-type.boschshc.communication-quality.state.option.BAD = Bad
+channel-type.boschshc.communication-quality.state.option.MEDIUM = Medium
+channel-type.boschshc.communication-quality.state.option.NORMAL = Normal
+channel-type.boschshc.communication-quality.state.option.GOOD = Good
+channel-type.boschshc.communication-quality.state.option.UNKNOWN = Unknown
```

Comment:
These are no longer relevant, correct?

Reply:
You are correct, I forgot to re-generate the i18n file again 😉